test/distributed/org/apache/cassandra/distributed/test/JavaDriverUtils.java

```diff
@@ -77,4 +78,24 @@ public static List<InetSocketAddress> buildContactPoints(ICluster<? extends IIns
                     .map(ClusterUtils::getNativeInetSocketAddress)
                     .collect(Collectors.toList());
     }
+
+    public static ConsistencyLevel toDriverCL(org.apache.cassandra.distributed.api.ConsistencyLevel cl)
```

Comment:
This is unused here?

Reply:
not used anymore.  This was moved out of `StatefulASTBase`.  In trunk i have a test to repo a history, so had to refactor that out